pkg/schema.go

```diff
@@ -61,10 +73,81 @@ func (s TSSchema[T]) GetMeasureNameFor(metricName T) (string, error) {
 
 // GetTableNameFor retrieves the table name where the given metric name is
 // stored. If the metric name is not found, it returns an error.
-func (s TSSchema[T]) GetTableNameFor(metricName T) (string, error) {
+func (s TSSchema[T1, T2]) GetTableNameFor(metricName T2) (string, error) {
 	v, ok := s.invertedSchema[metricName]
 	if !ok {
 		return v.tableName, fmt.Errorf("metric name %T not found", metricName)
 	}
 	return v.tableName, nil
 }
+
+type PredefinedValues[T comparable] map[T]float64
+
+// GenerateDummyData generates dummy data based on the schema structure.
+// This function now uses the telemetry.MetricName type for keys.
```

Comment:
I don't think `telemetry.MetricName` is a valid type here right?

Reply:
correct, this needs to go